src/client/Microsoft.Identity.Client/TokenSource.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Specifies the source of the access and Id tokens in the authentication result.
+    /// </summary>
+    public enum TokenSource
+    {
+        /// <summary>
+        /// The source of access and Id token is not set.
+        /// </summary>
+        None = 0,
```

Comment:
It looks like you are using this value for internal logic, but it does not have any meaning in the public API. There is always a source for the token. Let's not expose this.

Reply:
I was using it for internal logic but also I wanted to keep default to some other value in case the token source is not set or missed in some flow. Maybe rename None to Unidentified to make it more relevant?

Reply:
Removed.